Monika After Story/game/script-ch30.rpy

```diff
@@ -474,6 +479,30 @@ label ch30_autoload:
     #If you were interrupted, push that event back on the stack
     $restartEvent()
 
+    #Grant XP for time spent away from the game if Monika was put to sleep right
+    python:
+        if len(persistent.sessions)>=2 and persistent.closed_self:
+            away_experience_time=time.time()-persistent.sessions[-2][1] #Time since end of previous session
+            away_xp=0
+
+            #Reset the idlexp total if monika has had at least 6 hours of rest
+            if away_experience_time >= 6*360:
```

Comment:
Could you make the 6 hour value a constant somewhere? Since this is going to be like a criteria for if a day has passed, `jokes-concept` will rely on this for daily metrics. 

In addition, other daily things could rely on this value, and having it defined once makes it easier to adjust in the future.

I recommend constants be located in their own `stores` to avoid overwriting them.